prismic-model/js/exhibitions.js

```diff
@@ -9,13 +9,17 @@ import link from './parts/link';
 import number from './parts/number';
 import list from './parts/list';
 import text from './parts/text';
+import select from './parts/select';
+import structuredText from './parts/structured-text';
 
 const Exhibitions = {
   Exhibition: {
+    format: select('Format', ['Permanent', 'Temporary']),
```

Comment:
For event series we actually create a different type - given we're trying to avoid tech creep, it might be good to keep it the same...

https://github.com/wellcometrust/wellcomecollection.org/blob/master/prismic-model/json/event-formats.json